src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -71,25 +71,30 @@ public override ExportResult Export(in Batch<Activity> batch)
                             continue;
                         }
 
-                        var array = tag.Value as Array;
-
-                        if (array == null)
+                        if (tag.Value is not Array array)
                         {
                             this.WriteLine($"    {tag.Key}: {tag.Value}");
                             continue;
                         }
 
                         this.WriteLine($"    {tag.Key}: [{string.Join(", ", array.Cast<object>())}]");
                     }
+                }
 
-                    if (!string.IsNullOrEmpty(statusCode))
+                if (activity.Status != ActivityStatusCode.Unset)
+                {
+                    this.WriteLine($"StatusCode : {activity.Status}");
+                    if (!string.IsNullOrEmpty(activity.StatusDescription))
                     {
-                        this.WriteLine($"   StatusCode : {statusCode}");
+                        this.WriteLine($"Error : {activity.StatusDescription}");
```

Comment:
Remove "Error". Just Status Description is good enough here.

Reply:
Maybe something like:
`this.WriteLine($"Status Description : {activity.StatusDescription}");`
?

Reply:
yes

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3064